pgadmin_servers.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Servers": {
```

Comment:
Might be good to make this an `.example` like the `.env.example` above to allow for users that don't use the default `postgres` values for db name, username, and password. Then we can add `pgadmin_servers.json` to the .gitignore, and commit the example so users can fill in their own values.